airflow/providers/amazon/aws/operators/glue_session.py

```diff
@@ -0,0 +1,206 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from functools import cached_property
+from typing import TYPE_CHECKING, Any, Sequence, Union
+
+from airflow.configuration import conf
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.providers.amazon.aws.hooks.glue_session import GlueSessionHook
+from airflow.providers.amazon.aws.triggers.glue_session import GlueSessionReadyTrigger
+from airflow.providers.amazon.aws.utils import validate_execute_complete_event
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class GlueCreateSessionOperator(BaseOperator):
```

Comment:
Please consider to use [`airflow.providers.amazon.aws.operators.base_aws.AwsBaseOperator`](https://github.com/apache/airflow/blob/6029c71e2c4a8f6bb7086728f4bc09f43d18c11c/airflow/providers/amazon/aws/operators/base_aws.py#L32)

Some examples:
- https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/appflow.py
- https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/athena.py
- https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/bedrock.py

Issue for the reference: https://github.com/apache/airflow/issues/35278